notebooks/224-stable-diffusion-text-to-image/224-stable-diffusion-text-to-image.ipynb

```diff
@@ -0,0 +1,1209 @@
+    "                image = image.transpose(0, 2, 3, 1)\n",
+    "                if output_type == \"pil\":\n",
+    "                    image = self.numpy_to_pil(image)\n",
+    "                img_buffer.extend(image)\n",
+    "\n",
+    "        # scale and decode the image latents with vae\n",
+    "        latents = 1 / 0.18215 * latents\n",
+    "        image = self.vae(latents)[self._vae_output]\n",
+    "\n",
+    "        image = (image / 2 + 0.5).clip(0, 1)\n",
+    "        image = image.transpose(0, 2, 3, 1)\n",
+    "        if output_type == \"pil\":\n",
+    "            image = self.numpy_to_pil(image)\n",
+    "\n",
+    "        return {\"sample\": image, 'iterations': img_buffer}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Configure Inference Pipeline"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "First of all, we should create instance of our OpenVINOModel wrapper. We use it for more convinient getting inference results."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 59,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from openvino.runtime import Core\n",
+    "core = Core()\n",
+    "text_enc = core.compile_model(TEXT_ENCODER_OV_PATH, 'CPU')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 60,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "unet_model = core.compile_model(UNET_OV_PATH, 'CPU')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 61,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "vae = core.compile_model(VAE_OV_PATH, 'CPU')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Besides, models tokenizer and scheduler are also important parts of pipeline. Let's define them and put all components together"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "from transformers import CLIPTokenizer\n",
+    "from diffusers.schedulers import LMSDiscreteScheduler\n",
+    "\n",
+    "lms = LMSDiscreteScheduler(\n",
+    "    beta_start=0.00085, \n",
+    "    beta_end=0.012, \n",
+    "    beta_schedule=\"scaled_linear\"\n",
+    ")\n",
+    "tokenizer = CLIPTokenizer.from_pretrained('openai/clip-vit-large-patch14')\n",
+    "\n",
+    "ov_pipe = OVStableDiffusionPipeline(\n",
+    "    tokenizer=tokenizer,\n",
+    "    text_encoder=text_enc,\n",
+    "    unet=unet_model,\n",
+    "    vae=vae,\n",
+    "    scheduler=lms\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now, we are ready to define text prompt for generation and run inference pipeline. \n",
+    "Optionally, you can also change random generator seed for latent state initialization and number of steps.\n",
+    "> **Note**: Consider increasing `steps` to get more precise results. A suggested value is `50`, as it will take longer time to process.\n"
```

Comment:
```suggestion
    "> **Note**: Consider increasing `steps` to get more precise results. A suggested value is `50`, but it will take longer time to process.\n"
```